x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -55,6 +55,7 @@ export const signalRulesAlertType = ({
         index,
         filters,
         language,
+        maxSignals,
```

Comment:
good idea to just pull this out of params instead of having `params.maxSignals`.